### ISSUE TITLE
gulp build-search failed

### ISSUE THREAD

**Issue by stylersnico** [Kikobeats/uno-urban#12]
Hi,

I have a problem with the <code>gulp build-search</code> command.

```
/var/www/ghost/abyssproject.net/content/themes/uno-urban$ gulp build-search
[11:44:37] Requiring external module coffee-script/register
[11:44:39] Using gulpfile /var/www/ghost/abyssproject.net/content/themes/uno-urban/gulpfile.coffee
[11:44:39] Starting 'build-search'...
[11:44:39] Finished 'build-search' after 7.92 ms
module.js:427
    throw err;
    ^

SyntaxError: /var/www/ghost/abyssproject.net/content/themes/uno-urban/search.json: Unexpected string
    at Object.parse (native)
    at Object.Module._extensions..json (module.js:424:27)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/var/www/ghost/abyssproject.net/content/themes/uno-urban/node_modules/algolia-webcrawler/app.js:10:14)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
```

How can I solve this?

Thanks

**Comment by Kikobeats** [comment]
That was my fault because I release a version with `search.example.json` malformed 😅 .

It is fixed in the last `2.9.0` version. Please, download this last version and create again your `search.json`.

If you need help feel free to reply!

**Comment by stylersnico** [comment]
Yes fixed ;)

Thanks
